exercises/secret-handshake/src/example/kotlin/Signal.kt

```diff
@@ -0,0 +1,5 @@
+enum class Signal {
```

Comment:
Ah, yeah, Java was doing the same thing but that's not Windows-friendly. Happy to implement either option, let me know which you'd prefer!